CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ generate_messages(DEPENDENCIES std_msgs geometry_msgs)
 
 #add catkin sourced packages:
 catkin_package(
-   LIBRARIES ${PROJECT_NAME}
```

Comment:
The other libraries built by this package should be listed here, though:

```
    LIBRARIES dynamics_manager keyboard_op door_plugin elevator auto_door
```

Reply:
Added the 3 libraries to the catkin package definition :)